### DISCUSSION TITLE
Labs Feedback: @lit-labs/motion

### DISCUSSION THREAD

**Discussion by sorvell** [lit/lit#3351] [Labs Feedback]
The `@lit-labs/motion` package provides Lit directives designed to support easily creating simple animations powered by the web animations API under the hood. The `animate` directive can be used to animate DOM elements from one render to the next. If the animate element changes state between renders, the directive performs a "tweening" animation between the two states based on the options given. In addition, elements can animate when they initially render and when they are removed.

We want to collect your feedback on your experiences with `@lit-labs/motion` as we iterate on it according to our Lit Labs process and move the package towards possible graduation or retirement.

- Status: **Under Development**
- Major known issues:
  - https://github.com/lit/lit/issues/3016
  - https://github.com/lit/lit/issues/2903
- Issues: [[labs/motion]](https://github.com/lit/lit/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+%5Blabs%2Fmotion%5D)
- Docs: [README](https://github.com/lit/lit/tree/main/packages/labs/motion#readme)
- npm: [@lit-labs/motion](https://www.npmjs.com/package/@lit-labs/motion)

Please try `@lit-labs/motion` and tell us about your experience. Leave comments with your thoughts; highlight what issues or missing features you're blocked on; give a link to a project that uses it; etc.

And so we can have some numbers to look at, you can add reactions to this comment to indicate your experience with this package:

🚀 I have tried this package and it works great. I need it. Ship it!
👍 I have tried this package and it seems like a good idea.
😕 I have tried this package and would use it, but it has serious bugs or missing features.
👀 I am interested, but haven't tried the package yet.
👎 I'm not interested in this package.

Note: other reactions like 😄, 🎉, and ❤️ don't mean anything as part of this scheme. You probably mean, and please use, 🚀, 👍, or  👀.

**Comment by yorrd** [comment]
I have voted on a number of these, and this comment applies to all of them, but especially to the `@lit-labs/motion` idea. We are currently writing our own functionality which we have started when you first released lit. Ever since then, we've been wondering when you're going to graduate all these ideas and would be on board within a week with the whole team. This applies to ssr, routing, motion, react wrapping, etc, and I have upvoted all these individually. 

All this is much appreciated and we're ready to buy in to a broader ecosystem which you're starting to create here. Webcomponents are the solution and lit is the only well-backed library that does it without weird custom transpilers or magic strings. We're using as much of lit as possible in our engine and for all customer projects. Please keep this going <3
